openlibrary/plugins/upstream/borrow.py

```diff
@@ -123,19 +123,25 @@ def POST(self, key):
         # to result if `open`, redirect to bookreader
         response = lending.get_availability_of_ocaid(edition.ocaid)
         availability = response[edition.ocaid] if response else {}
+        archive_url = 'https://archive.org/stream/' + edition.ocaid + '?ref=ol'
+        if i._autoReadAloud is not None:
+            archive_url += '&_autoReadAloud=show'
```

Comment:
Not a fan of this key, val pair -- no intention of blocking this decision :)
(a) `auto` is confusing
(b) using `show` seems like it could lead to a hacky paradigm decision (wherein many features try to show/hide, forcing us to check a lot of different state cases

Reply:
_ is meant for private access; we can rename it in the future. the purpose of show was that if we wanted to have `autoReadAloud=start` (which autoplays or tries to).

Reply:
won't fix